pkgs/applications/misc/elektroid/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk3
+, wrapGAppsHook
+, autoreconfHook
+, pkg-config
+, gettext
+, json-glib
+, automake
+, autoconf
+, libtool
+, alsa-lib
+, libpulseaudio
+, libsndfile
+, libsamplerate
+, zlib
+, libjson
+, libzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "elektroid";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "dagargo";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-T7jxtmxlFe5WPC2KYDN6jDHwuVkmA6Y/h9qizZFH5rk=";
+  };
+
+  buildInputs = [
+    wrapGAppsHook
+    autoreconfHook
+    gtk3
+    pkg-config
+    gettext
+    json-glib
+    automake
+    autoconf
+    libtool
```

Comment:
Should some of these go to nativeBuildInputs? https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/build-tools-in-build-inputs.md